SU2_CFD/src/driver_structure.cpp

```diff
@@ -119,45 +123,61 @@ CDriver::CDriver(char* confFile,
 
     config_container[iZone]->SetMPICommunicator(MPICommunicator);
 
-    /*--- Definition of the geometry class to store the primal grid in the
+    /*--- Read the number of instances for each zone ---*/
+
+    nInst[iZone] = config_container[iZone]->GetnTimeInstances();
```

Comment:
Any particular motivation for the index ordering, i.e., geometry[iZone][iInst] vs. geometry[iInst][iZone]? Any important implications for the loops in the driver, for example?